### ISSUE TITLE
Investigate and resolve CircleCI test failures

### ISSUE THREAD

**Issue by rpkyle** [plotly/dash-bio#514]
There are currently multiple recurring CircleCI integration test failures for `Clustergram` and `Ideogram`, even when files in the repository (such as the CircleCI `config.yml`) unrelated to the components themselves are modified, e.g. within the `python-3.7` workflow:

```python
Results (484.15s):
      48 passed
       6 failed
         - tests/integration/test_clustergram.py:22 test_dbcl001_colorscale
         - tests/integration/test_clustergram.py:47 test_dbcl002_cluster_by_row_or_col
         - tests/integration/test_clustergram.py:95 test_dbcl003_row_col_thresholds
         - tests/integration/test_clustergram.py:126 test_dbcl004_col_annotations
         - tests/integration/test_clustergram.py:158 test_dbcl005_row_annotations
         - tests/integration/test_clustergram.py:190 test_dbcl006_df_input_row_cluster
```

... and the `python3.6` workflow as well:

```python
Results (526.74s):
      47 passed
       7 failed
         - tests/integration/test_clustergram.py:22 test_dbcl001_colorscale
         - tests/integration/test_clustergram.py:47 test_dbcl002_cluster_by_row_or_col
         - tests/integration/test_clustergram.py:95 test_dbcl003_row_col_thresholds
         - tests/integration/test_clustergram.py:126 test_dbcl004_col_annotations
         - tests/integration/test_clustergram.py:158 test_dbcl005_row_annotations
         - tests/integration/test_clustergram.py:190 test_dbcl006_df_input_row_cluster
         - tests/integration/test_ideogram.py:13 test_dbid001_displayed_chromosomes
```

We should sort out why these tests are failing and get them to pass again.

**Comment by alexcjohnson** [comment]
I believe all the clustergram ones should be addressed by #512 (possibly with some test adjustments after we get the tests running again in that PR).

Is the ideogram one consistent or sporadic?

**Comment by rpkyle** [comment]
Thanks to @nicholas-esterer 🥇, I think nearly all the test failures in `dash-bio` have been addressed (I contributed very minor fixes to address the `babel` installation issue, as well as the `.pylintrc` change to silence the `bad-continuation` warnings. 

I'll close this one, since the remaining Percy issue described in #474 already has its own issue. @HammadTheOne and I will work to sort that one out when time permits, post-release.
